common/static/css/vendor/pdfjs/viewer.css

```diff
@@ -1988,3 +1988,10 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
     display: none;
   }
 }
+
+.toolbarButton#openFile,
+.toolbarButton#download,
+.secondaryToolbarButton#secondaryOpenFile,
+.secondaryToolbarButton#secondaryDownload {
+    display: none;
+}
```

Comment:
You should configure your text editor to insert newlines at EOF.

(and/or at least do it manually here)